graphql/executor.go

```diff
@@ -283,6 +287,14 @@ func (e *Executor) execute(ctx context.Context, typ Type, source interface{}, se
 	switch typ := typ.(type) {
 	case *Scalar:
 		return unwrap(source), nil
+	case *Enum:
+		val := unwrap(source)
+		for key, mapVal := range typ.Mapping {
```

Comment:
could you precompute this reverse map? having to enumerate here could be expensive for enums with lots of values.